source/splitSingletrack/mod_splitSingletrack.f90

```diff
@@ -0,0 +1,210 @@
+module mod_splitSingletrack
+  use iso_fortran_env, only : output_unit
+  implicit none
+
+  ! File units used for I/O;
+  ! can be overridden USEing if clashing
+  ! with units used in rest of the code.
+  integer :: inUnit = 90
+  integer :: outUnit = 91
+
+  integer :: myOutUnit = output_unit
```

Comment:
The output_unit variable is used in the module so that it is easyer to integrate with the top-level SixTrack. I could get the iso_fortran_env::output unit in the program and use that (and set it for the module), but it would not make any difference.

I'll change the variable name.

Reply:
Would probably make more sense to just call it lout then, but ok.